### PR TITLE
fix: reset translation branch to master

### PR DESCRIPTION
**Pull request by sparten11740**
## Summary
The action currently keeps failing because it cannot resolve the rebase issues. This wipes the branch clean to be the same as master before executing the script.

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -21,93 +21,86 @@
     "@octokit/plugin-rest-endpoint-methods" "^5.13.0"
 
 "@actions/http-client@^2.0.1":
-  version "2.0.1"
-  resolved "https://registry.yarnpkg.com/@actions/http-client/-/http-client-2.0.1.tgz#873f4ca98fe32f6839462a6f046332677322f99c"
-  integrity sha512-PIXiMVtz6VvyaRsGY268qvj57hXQEpsYogYOu2nrQhlf+XCGmZstmuZBbAybUl1nQGnvS1k1eEsQ69ZoD7xlSw==
+  version "2.1.0"
```
Comment (diegomura) [inline yarn.lock:24]: Why all these yarn.lock changes? Cannot force these changes locally

Comment (sparten11740) [reply yarn.lock]: it didn't fix the issue, reverted the lockfile

Comment (sparten11740) [reply yarn.lock]: (your last push to master has fails with the same problem on install: `Invariant Violation: Expected a key`)

Comment (sparten11740) [reply yarn.lock]: will PR with a different fix

**Pull request closed without merging**